### DISCUSSION TITLE
Is based on vulkan

### DISCUSSION THREAD

**Discussion by alavaelian** [slint-ui/slint#1626] [Q&A]
Or based in opengl

**Comment by hunger** [comment]
Right now: GL or pure software rendering (either framebuffer or linebuffer based or via Qt Painter).

We recently had some progress on DirectX and Metal as backends as well, but I did not follow that topic and can not comment on the state of those efforts.

We also are working on a Skia rendering backend (not production ready yet!), which at least mentions Vulkan support in its manual under "Special builds".

Slint has lots of options to scale UIs well from tiny devices all the way up to powerful graphics hardware. Any help in enabling new hardware is always welcome :-)

**Reply by tronical** [reply]
Tobias is right. We plan on enabling Vulkan based rendering through our new Skia renderer. That is mostly (massive amounts of) glue code to connect ash and Skia. https://github.com/slint-ui/slint/issues/1546 is tracking this - help is welcome if you have Vulkan experience!

**Comment by alavaelian** [comment]
Witch would be fastest ? With vulkan or without it

**Reply by tronical** [reply]
That is difficult to predict reliably.
